library/openssl/digest/digest_length_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../../../spec_helper'
+require_relative '../../../library/digest/sha1/shared/constants'
+require_relative '../../../library/digest/sha256/shared/constants'
+require_relative '../../../library/digest/sha384/shared/constants'
+require_relative '../../../library/digest/sha512/shared/constants'
+require 'openssl'
+
+describe "OpenSSL::Digest#digest_length" do
+  describe "digest_length of OpenSSL::Digest.new(name)" do
+    it "returns a SHA1 block length" do
```

Comment:
Is `block length` a typo here?

Reply:
More likely an oversight from a copy-paste